lib/expand/join.go

```diff
@@ -830,7 +830,8 @@ func (p *Peer) tryConnect(operationID string) (ctx *operationContext, err error)
 			return nil, utils.Abort(err)
 		}
 		if trace.IsCompareFailed(err) {
-			p.printStep("Waiting for another operation to finish at %v", addr)
+			p.WithError(err).Info("Failed to create expand operation.")
+			p.printStep("Waiting to create expand operation at %v: %v", addr, err)
```

Comment:
So I'm not sure how this displays when encountered, but if the error text sounds like an error I could see it causing confusion on how to proceed, even though it's likely just waiting for the cluster to complete it's operation.

Reply:
Make sense - I'll provide a couple of deviations:
```
Waiting for cluster to become healthy
```
when the cluster is degraded and
```
Waiting for another operation to complete
```
for other cases (`cannot create operation for master with another node joining`, `cannot create operation for node while master is joining`, `join capacity is exceeded`)